Error: The WASM interface has not been initialized

### Context

Happened multiple times recently:

https://github.com/litentry/litentry-parachain/actions/runs/4247565225
https://github.com/litentry/litentry-parachain/actions/runs/4250721399

Before any ts-tests (so not ts-tests problem):
```
Trace: Error: Error: The WASM interface has not been initialized. Ensure that you wait for the initialization Promise with waitReady() from @polkadot/wasm-crypto (or cryptoWaitReady() from @polkadot/util-crypto) before attempting to use WASM-only interfaces.
    at /home/runner/work/litentry-parachain/litentry-parachain/docker/node_modules/@polkadot/wasm-crypto/cjs/bundle.js:46:13
    at sr25519PairFromSeed (/home/runner/work/litentry-parachain/litentry-parachain/docker/node_modules/@polkadot/util-crypto/cjs/sr25519/pair/fromSeed.js:22:82)
    at Object.sr25519 (/home/runner/work/litentry-parachain/litentry-parachain/docker/node_modules/@polkadot/keyring/cjs/keyring.js:19:56)
    at Keyring.createFromUri (/home/runner/work/litentry-parachain/litentry-parachain/docker/node_modules/@polkadot/keyring/cjs/keyring.js:256:115)
    at generateRelaychainGenesisFile (/home/runner/work/litentry-parachain/litentry-parachain/docker/node_modules/@open-web3/parachain-launch/lib/index.js:1[43](https://github.com/litentry/litentry-parachain/actions/runs/4250721399/jobs/7393584955#step:8:44):31)
    at generate (/home/runner/work/litentry-parachain/litentry-parachain/docker/node_modules/@open-web3/parachain-launch/lib/index.js:374:5)
    at fatal (/home/runner/work/litentry-parachain/litentry-parachain/docker/node_modules/@open-web3/parachain-launch/lib/index.js:55:13)
make: *** [Makefile:175: generate-docker-compose-rococo] Error 1
```

Is it a parachain-launch problem? We use [parachain-launch](https://github.com/open-web3-stack/parachain-launch) to start the local parachain network with docker.

---
:heavy_check_mark: Please set appropriate **labels** and **assignees** if applicable.

--- a/.github/file-filter.yml
+++ b/.github/file-filter.yml
@@ -8,7 +8,6 @@ parachain_src: &parachain_src
   - 'runtime/**'
   - 'mock-tee-primitives/**'
   - 'docker/Dockerfile'
-  - 'docker/bridge.dockerfile'
   - 'Cargo.lock'
   - 'Cargo.toml'
   - 'rust-toolchain.toml'
@@ -33,3 +32,4 @@ tee_test: &tee_test
   - 'tee-worker/ts-tests/**'
   - 'tee-worker/docker/*.yml'
   - 'tee-worker/cli/*.sh'
+  - 'docker/**'

--- a/.github/file-filter.yml
+++ b/.github/file-filter.yml
@@ -15,6 +15,7 @@ parachain_src: &parachain_src
 
 parachain_test: &parachain_test
   - 'ts-tests/**'
+  - 'docker/**'
 
 coverage_src: &coverage_src
   - *parachain_src

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -375,7 +375,7 @@ jobs:
         uses: andymckay/cancel-action@0.3
 
   tee-test:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     needs:
       - check-file-change
       - parachain-build

--- a/.gitignore
+++ b/.gitignore
@@ -8,7 +8,6 @@
 **/node_modules/
 
 docker/yarn.lock
-docker/package.json
 docker/generated-*/
 tee-worker/docker/litentry/
 

--- a/docker/package.json
+++ b/docker/package.json
@@ -7,6 +7,6 @@
     "start": "node_modules/@open-web3/parachain-launch/bin/parachain-launch"
   },
   "dependencies": {
-    "@open-web3/parachain-launch": "^1.2.2"
+    "@open-web3/parachain-launch": "^1.3.0"
   }
 }

--- a/docker/package.json
+++ b/docker/package.json
@@ -1,0 +1,12 @@
+{
+  "name": "litentry-parachain-launch",
+  "version": "1.0.0",
+  "main": "",
+  "license": "GPL-3.0",
+  "scripts": {
+    "start": "node_modules/@open-web3/parachain-launch/bin/parachain-launch"
+  },
+  "dependencies": {
+    "@open-web3/parachain-launch": "^1.2.2"
+  }
+}

--- a/scripts/generate-docker-files.sh
+++ b/scripts/generate-docker-files.sh
@@ -14,8 +14,6 @@ function print_divider() {
 ROOTDIR=$(git rev-parse --show-toplevel)
 cd "$ROOTDIR/docker"
 
-PARCHAIN_LAUNCH_BIN="$ROOTDIR/docker/node_modules/.bin/parachain-launch"
-
 CHAIN=$1
 CONFIG=$CHAIN-parachain-launch-config.yml
 OUTDIR=generated-$CHAIN
@@ -23,7 +21,7 @@ OUTDIR=generated-$CHAIN
 print_divider
 
 echo "installing parachain-launch ..."
-yarn add -s @open-web3/parachain-launch
+yarn
 print_divider
 
 # pull the polkadot image to make sure we are using the latest
@@ -34,7 +32,7 @@ docker pull -q "$POLKADOT_IMAGE"
 
 print_divider
 
-"$PARCHAIN_LAUNCH_BIN" generate --config="$CONFIG" --output="$OUTDIR" --yes
+yarn start generate --config="$CONFIG" --output="$OUTDIR" --yes
 
 echo "Done, please check files under $ROOTDIR/docker/$OUTDIR/"
 print_divider

--- a/scripts/generate-docker-files.sh
+++ b/scripts/generate-docker-files.sh
@@ -32,6 +32,7 @@ docker pull -q "$POLKADOT_IMAGE"
 
 print_divider
 
+yarn start --version
 yarn start generate --config="$CONFIG" --output="$OUTDIR" --yes
 
 echo "Done, please check files under $ROOTDIR/docker/$OUTDIR/"

